renovate.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "driver_config.yml"
+      ],
+      "matchStrings": [
+        "cuda:\\s*(?:#.*)?\\n\\s*version:\\s*\"(?<currentValue>[^\"]+)\""
+      ],
+      "depNameTemplate": "nvidia-cuda-driver",
+      "datasourceTemplate": "custom.nvidia-driver",
+      "versioningTemplate": "loose"
+    }
+  ],
+  "customDatasources": {
+  "nvidia-driver": {
+    "defaultRegistryUrlTemplate": "https://docs.nvidia.com/datacenter/tesla/drivers/releases.json",
+    "transformTemplates": [
+      "{ \"releases\": $map($reduce($map($keys($), function($k) { { \"key\": $k, \"value\": $lookup($, $k) } }), function($acc, $v) { $append($acc, $v.value.driver_info) }, []), function($info) { { \"version\": $info.release_version } }) }"
+    ]
+  }
+ },
+  "packageRules": [
+    {
+      "excludePackagePatterns": [".*"],
+      "managers": ["regex"],
+      "matchManagers": ["regex"],
+      "enabled": true
+    }
+  ],
+  "logLevelRemap": [
```

Comment:
do we want to keep this around ?

Reply:
I think keeping till we see regular updates for cuda drivers would be helpful. Now, this repo hasn't got approval to use renovate yet so it's not been fully tested.